File: javascript/challenge10.js
const whereCanIPark = (spots, vehicle) => {
  let OPTIONS = { regular : 'R', small: 'RS', motorcycle: 'RSM' }
  for (let n = 0; n < spots.length; n++){
    for (let m = 0; m < spots[n].length; m++){
      if (OPTIONS[vehicle].includes(spots[n][m])){
        return [m, n] 
      }
    }
  }
  return false
}

File: python/challenge10.py
def whereCanIPark(spots, vehicle):
    options = { "regular" : "R", "small": "RS", "motorcycle": "RSM" }
    for n in range(0, len(spots)):
        for m in range(0, len(spots[n])):
            if options[vehicle].__contains__(spots[n][m]):
                return [m, n]
    return False

# vehicle = "regular"
# spots = [['s', 's', 's', 'S', 'R', 'M'], ['s', 'M', 's', 'S', 'R', 'M'], ['s', 'M', 's', 'S', 'R', 'm'], ['S', 'r', 's', 'm', 'R', 'M'], ['S', 'r', 's', 'm', 'R', 'M'],    ['S', 'r', 'S', 'M', 'M', 'S']]
# whereCanIPark(spots, vehicle)
